### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by prayag17** [jellyfin/jellyfin.org#7]
Hello, I saw that you guys are rebuilding the site and docs, I wanted to help, so I made a mockup for the site. 

New Design Mockup:
![JF Site design](https://user-images.githubusercontent.com/55829513/126743855-13ea90dd-4d0b-4c07-9100-ccdb0d7d17cd.png)

The hero page will have a particle js canvas(can be removed).

I know there are some alignment issues but that will be fixed in the html.

I can make this on my own f you guys like it. if you guys like it comment below and I will start making it.

I now understand that a static site like this should have minimal JS so I will try to keep the JS animation to minimal(2-3 at most, that won't break the site even if js is doesn't load).

**Comment by MrTimscampi** [comment]
I've been away from this repo for a bit due to a job change, some stuff over on jf-vue and other real life stuff, so thanks for looking at it :)

There are some parts I like about the changes and some I don't.

What I don't like:

* The particles effect
  I find it incredibly generic and not related to Jellyfin or media at all. It's standard "tech-looking" effect that I don't think has its place here.
  I would really like something like the Plex header, but I'm not sure about rights here, and we very much want to distance ourselves from copyrighted content in advertising ourselves:
![image](https://user-images.githubusercontent.com/19396809/126767995-025feaac-7666-4b2b-ad82-588c937b247c.png)
* Repeating "Jellyfin" in the hero title. I don't think it's necessary, since the logo is right above.
* The transition from a hard cut on the hero block to a color gradient, then all color gradients until the end, followed by a hard cut for the CTA block.
* The change of color for the Undraw images. I know they're purple on the current website, but the change of color here was deliberate. Essentially, I wanted to press more on our primary color, and have slightly more... contrast? with the background color (It's not the right word, but it conveys the idea).
  The main issue with our current color design is that we're forced into a fairly monochrome scheme, due to our primary having really crappy and jarring complimentary colors (Brown for normal complimentary and orange+yellow for split complementary).
  Purple looks okay with blue, but for the Undraw icons, I think it clashes too much. It's debatable though, and I'd love to get the opinion of a proper graphic designer on it.
* The outlined icons in the client section look out of place. The Roku one is completely different from the rest and some like the Amazon icon look plain weird. We're also using Simple Icons for these, for simplicity's sake, and I don't think they modify the brand icons in any way (And doing so may be an issue, since, you know, trademarks, brand usage guidelines and all, which we have to stay aware of when we use brand names).

What I like:

* The gradient on the text for the info sections. It looks nice.
* The backdrop icons for the info sections. It has a nice style. Though I was going for something like this (albeit limited by our really shitty color palette):
![image](https://user-images.githubusercontent.com/19396809/126771426-ce8a6564-d3a3-4a90-b067-5cb76cc58972.png)
* The footer change, though I would use a copy of the full logo at the same size and alignment as the top bar, for consistency (Which will need changes on the bar, which we should do anyway. It shouldn't be taking the entire width, that just looks weird)

---

Side note, this new website, while statically built (So all the content is pre-generated) uses React, so it's all in Javascript (Though it works without it).

The reason for this is that we wanted to be able to have more dynamic pages for some parts of the website, notably clients and plugins.  
The idea there would be to build both of these dynamically from a bunch of JSON files (The official plugin manifest for the latter, and a JSON file contained in this repository for the former).

It's not directly related to this PR, but since you mention JS, I thought it'd be important to mention that everything needs to be React-compliant (There are a bunch of linting rules on the repo already, if I'm not mistaken).

**Comment by prayag17** [comment]
The blocky transition of hero bg and brief's bg is because of the particle BG and I was gonna remove that with a mask but since we are not doing it then it is not required.

I agree with you about the color palette too, I dont like it. I did try to use it in my CSS and it works well there but I don't like it that much. I did create a custom color palette but didn't want to change the default color palette of the site as I was afraid that other Dev's won't like.

By JS here I meant JS animations like lazy loading
